internal/support/scheduler/schedulecontext.go

```diff
@@ -70,7 +67,11 @@ func (sc *IntervalContext) Reset(interval models.Interval) {
 	//frequency and next time
 	nowBenchmark := time.Now().Unix()
 	if !sc.Interval.RunOnce {
-		sc.Frequency = parseFrequency(sc.Interval.Frequency)
+		frequency, err := parseFrequency(sc.Interval.Frequency)
+		if err != nil {
+			LoggingClient.Error("parse frequency error, the original frequency string is : " + sc.Interval.Frequency)
```

Comment:
Right now, you're eating the error message.

Reply:
fixed . logging error message as well.